app/main.py

```diff
@@ -1,3 +1,119 @@
+            return Distance(self.km * other.km)
+
+        raise TypeError(
+            f"unsupported type for __mul__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __truediv__(self, other: int | float) -> Distance | TypeError:
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return Distance(round(self.km / other.km, 2))
+        raise TypeError(
+            f"unsupported type for __truediv__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __lt__(self, other: int | float) -> bool | TypeError:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return self.km < other.km
+        raise TypeError(
+            f"unsupported type for __lt__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __gt__(self, other: int | float) -> bool | TypeError:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return self.km > other.km
+        raise TypeError(
+            f"unsupported type for __gt__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __eq__(self, other: int | float) -> bool | TypeError:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return self.km == other.km
+        raise TypeError(
+            f"unsupported type for __eq__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __le__(self, other: int | float) -> bool | TypeError:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return self.km <= other.km
+        raise TypeError(
+            f"unsupported type for __le__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
+
+    def __ge__(self, other: int | float) -> bool | TypeError:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        if isinstance(other, (int, float)):
+            other = Distance(other)
+            return self.km >= other.km
+        raise TypeError(
+            f"unsupported type for __ge__ method. <class 'float'> "
+            f"or <class 'int'> were expected. "
+            f"But received {type(other)}"
+        )
```

Comment:
u can use previous methods here, and implement them in one line )

Reply:
you mean that I should user the ```__lt__``` and ```__gt__``` under ```__le__``` and ```__ge__```? cause I didn't rly understand